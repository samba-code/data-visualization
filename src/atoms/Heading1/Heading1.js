import React from 'react';
import styled from 'styled-components';
import { rem } from 'polished';

import PropTypes from 'prop-types';

const StyledHeading1 = styled.div`
    font-size: ${rem("24px")};
    color: ${(props) => props.theme.primary};
    margin: 20px;
`;

const Heading1 = ({children}) => <StyledHeading1>{children}</StyledHeading1>;

Heading1.propTypes = {
    children: PropTypes.string
}

export default Heading1;