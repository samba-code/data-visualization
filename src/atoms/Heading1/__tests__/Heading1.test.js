import React from "react";
import { render } from "enzyme";

import Heading1 from "../Heading1";

describe("Heading1", () => {
  it("should match snapshot", () => {
    const wrapper = render(<Heading1 />);
    expect(wrapper).toMatchSnapshot();
  });
});
