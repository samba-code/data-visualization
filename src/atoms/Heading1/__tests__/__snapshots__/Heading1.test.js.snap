// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading1 should match snapshot 1`] = `
<h1
  class="Heading1__StyledHeading1-f9ligb-0 kWDRoF"
/>
`;
